lib/src/products_page.dart

```diff
@@ -41,7 +47,39 @@ class _Products extends StatelessWidget {
 class _Product extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Product Item');
+    return Card(
+      child: Column(
+        children: <Widget>[
+          AspectRatio(
+            aspectRatio: 20.0 / 11.0,
```

Comment:
Actually let's use the aspectRatio param on the GridView:
    return GridView.builder(
      gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
        crossAxisCount: 2,
        childAspectRatio: 1 / 1.2
      ),

Reply:
Done commit 26ccfb62d95c765ae078e7a068c78eeeb9c23457

Reply:
Let's also use the following for the gridDelegate:
`        mainAxisSpacing: 8,
        crossAxisSpacing: 8,
`
This is to add a bit of spacing for the grid items.